<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Public Key / –û—Ç–∫—Ä—ã—Ç—ã–π –∫–ª—é—á</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_19">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>e:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_public_e"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_20">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>n:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="le_public_n"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Explicit text / –û—Ç–∫—Ä—ã—Ç—ã–π —Ç–µ–∫—Å—Ç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteExplicitText"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnEncrypt">
        <property name="text">
         <string>üîêEncrypted Text (–ó–∞—à–∏—Ñ—Ä–æ–≤–∞—Ç—å —Ç–µ–∫—Å—Ç)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Coded text / –®–∏—Ñ—Ä–æ–≤–∞–Ω–Ω—ã–π —Ç–µ–∫—Å—Ç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="tbCodedText"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSendCodedText">
        <property name="text">
         <string>Send coded text to Client ‚úà</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
